### ISSUE TITLE
Project thumbnail should have alt text

### ISSUE THREAD

**Issue by jhung** [inclusive-design/idrc#495]
## Is your feature request related to a problem? Please describe.

The project thumbnails for the projects listing should have alt text. Currently the authoring backend does not allow entering of alt-text.

## Describe the solution you'd like

Provide an alt-text field for the author to write alt text for the thumbnail image. If the alt-text field is left empty, upon submission the alt-text will be auto-generated using the Project Name using the following template: "Thumbnail image for [Project name]".
